jigasi-home/sip-communicator.properties

```diff
@@ -66,6 +66,75 @@ net.java.sip.communicator.impl.protocol.sip.acc1403273890647.DOMAIN_BASE=<<DOMAI
 # can be enabled to disable audio mixing and use translator, jigasi will act as jvb, just forward every ssrc stream it receives.
 #net.java.sip.communicator.impl.protocol.sip.acc1403273890647.USE_TRANSLATOR_IN_CONFERENCE=true
 
+# The following account is used only for dial-out from a meeting, it connects and join the brewery room used
+# by Jicofo to discover available jigasi instances and use them (they can be used for outgoing calls or transcriptions if enabled or both)
+net.java.sip.communicator.impl.protocol.jabber.acc-xmpp-1=acc-xmpp-1
+net.java.sip.communicator.impl.protocol.jabber.acc-xmpp-1.ACCOUNT_UID=Jabber:jigasi@auth.<<DOMAIN_BASE>>
+net.java.sip.communicator.impl.protocol.jabber.acc-xmpp-1.USER_ID=jigasi@auth.<<DOMAIN_BASE>>
+net.java.sip.communicator.impl.protocol.jabber.acc-xmpp-1.IS_SERVER_OVERRIDDEN=true
+net.java.sip.communicator.impl.protocol.jabber.acc-xmpp-1.SERVER_ADDRESS=127.0.0.1
+net.java.sip.communicator.impl.protocol.jabber.acc-xmpp-1.SERVER_PORT=5222
+#net.java.sip.communicator.impl.protocol.jabber.acc-xmpp-1.BOSH_URL=https://127.0.0.1/http-bind
+net.java.sip.communicator.impl.protocol.jabber.acc-xmpp-1.ALLOW_NON_SECURE=true
+#base64 AES keyLength:256 or 128
+net.java.sip.communicator.impl.protocol.jabber.acc-xmpp-1.PASSWORD=<<JIGASI_XMPPPWD>>
```

Comment:
Can we name this more clearly? JIGASI_XMPP_PASSWORD_BASE64?

Reply:
👍

Reply:
Sorry, one more instance of the old name left in the readme